spec/requests/api/v2/users_spec.rb

```diff
@@ -8,54 +8,103 @@
 
   it_behaves_like 'ApiKeyAuthenticatable'
 
-  context 'with multiple users' do
-    let(:swipecard_code) { '1234567' }
-    let(:user_barcode) { '2470041440697' }
-
-    before do
-      create_list(:user, 5)
-      create :user, swipecard_code: swipecard_code
-      create :user, barcode: 'ID41440E'
-    end
+  context 'with multiple Users' do
+    let(:users) { create_list(:user, 5) }
 
-    it 'sends a list of users' do
+    it 'responds with a success HTTP status code' do
       api_get base_endpoint
 
-      # test for the 200 status-code
       expect(response).to have_http_status(:success)
+    end
+
+    it 'responds with all the Users' do
+      api_get base_endpoint
 
-      # check to make sure the right amount of messages are returned
-      expect(json['data'].length).to eq(7)
+      expect(json['data'].length).to eq User.count
     end
 
-    it 'allows filtering of users by user_code with swipecard' do
-      api_get "#{base_endpoint}?filter[user_code]=#{swipecard_code}"
+    describe 'filtering' do
+      let(:user) { users[2] }
```

Comment:
test for new uuid filter?

Reply:
Wow yeah!  I had meant to put that in, but it slipped my radar